### ISSUE TITLE
HDD hibernation not work on J1900 model Ds918+ DS 7.2

### ISSUE THREAD

**Issue by hoangpro1234** [AuxXxilium/arc#694]
**Describe the bug**
Hi Author's ARC. I'm using bootloader 23.12.4a for J1900 model DS918+ Ds7.2. I used addon sleep HDD. Everything works well except the hard drive sleep feature. I hope you will find a way to fix this error. A great software for the community. Thank you very much.

**Screenshots**

![hibernation](https://github.com/AuxXxilium/arc/assets/22629366/f7363d09-6e42-4487-b75f-08471c04856b)

**Comment by AuxXxilium** [comment]
First there is no addon "sleep". if you use the "hibernation" addon and your bios settings are correct, there is no way that it won't work except you install docker, container or vmm.

**Comment by hoangpro1234** [comment]
> First there is no addon "sleep". if you use the "hibernation" addon and your bios settings are correct, there is no way that it won't work except you install docker, container or vmm.

Yes, I used hibernation addon. Before, I installed Ds 6, hibernation feature work well.

**Comment by AuxXxilium** [comment]
If you upgrade from DSM 6, it will not work. And DSM 6 has another partition layout for system data, that will cause a overflow and full space after some time in dsm 7.

**Comment by AuxXxilium** [comment]
And if i check your screenshot, it is working but it has incoming connection from somewhere or is doing background tasks.

**Comment by hoangpro1234** [comment]
Before installing DS 7, I formatted the hard drive, but one time I mistakenly installed DS on the HDD instead of on the SSD. I then removed the HDD and reinstalled it on the SSD. After installation, reinsert the HDD.
I doubt that the DS installation partition on the HDD will cause a conflict leading to automatic wake-up. I want to delete the partition where DS is installed on the HDD, is that possible?
